cmd/thor/main.go

```diff
@@ -435,7 +435,7 @@ func soloAction(ctx *cli.Context) error {
 		return errors.New("block-interval cannot be zero")
 	}
 
-	printSoloStartupMessage(gene, repo, instanceDir, apiURL, forkConfig, metricsURL, adminURL)
+	printStartupMessage2(gene, apiURL, "", metricsURL, adminURL)
```

Comment:
IMO Solo and Main are better names for the methods vs 1 and 2.

Reply:
That is true, I didn't read it through. Yeah they're ok. I think We can name these a bit better in the future tho.